main.tf

```diff
@@ -2,7 +2,7 @@ locals {
   thanos_routes = var.thanos_public_endpoints ? [cloudfoundry_route.thanos.id, cloudfoundry_route.thanos_internal.id] : [cloudfoundry_route.thanos_internal.id]
   postfix       = var.name_postfix != "" ? var.name_postfix : random_id.id.hex
   prometheus_config = templatefile("${path.module}/templates/prometheus.yml", {
-    alertmanagers = ["0.tf-alertmanager-${local.postfix}.apps.internal:9093"]
```

Comment:
the leading `0.` indicates a specific instance on the internal routing on CF.

I think this has been done on purpose to make sure it will always go to a single instance..  @loafoe is that correct?

Reply:
@jdelucaa @ScottGuymer correct. Promethues states there should be no load balancing in front of alertmanager. So once the alertmanager module gains that capability it should export the individual instances. Safest is even to deploy distinct alertmanager apps and have them clustered. Promethues should then have distinct routes to each of them.

TL;DR the `0.` is signifcant